packages/sdk-drop-in/src/lib/fetchTokenDev.ts

```diff
@@ -0,0 +1,21 @@
+import { getConfig } from '@/lib/ConfigLoader';
+import { createAPIClient } from '@monite/sdk-react';
+
+export async function fetchTokenDev() {
+  const { entity_user_id, client_id, client_secret, api_url } =
+    await getConfig();
+  const { api, requestFn } = createAPIClient();
+  return await api.auth.postAuthToken(
+    {
+      baseUrl: `${api_url}/v1`,
+      parameters: {},
```

Comment:
Are we forced to pass an empty object? Can we just remove it as it's seems redundant

Reply:
Unfortunately, because of Qraft's type-system `parameters` are required. I have ideas on simplifying such cases, but this requires architecture changes and will be in the next version.